getting-started/overview/external-tools.rst

```diff
@@ -0,0 +1,245 @@
+.. _external-tools:
+
+==============
+External tools
+==============
+
+Archivematica integrates a suite of free and open-source tools that allows users to process 
+digital objects from ingest to access based on the ISO-OAIS functional model. In addition 
+to the core software which is released under `GNU Affero General Public License
+(AGPL)`_, the following tools are bundled with Archivematica.
```

Comment:
I think this reads quite well, but when we look down the list we see a mix of tools we're using to do a range of things:

* The Framework we're built upon.
* The tools we're using to provide preservation information about files.
* The utilities that provide other connecting tissues such as 7zip for packing our AIPs. 

Perhaps that first line can be something like:

"`Archivematica is built upon open-source frameworks and services (e.g. Django, Gearman below), as well as integrating a suite(s) of free and open-source tools that...`"

I'm not sure if you'd want to update `getting-started/overview/technical.rst` but it could potentially be added there too, or simplified in one of these two places to avoid redundancy.